README.md

```diff
@@ -30,7 +30,7 @@ description: Control the device status bar.
 StatusBar
 ======
 
-> The `StatusBar` object provides some functions to customize the iOS and Android StatusBar.
+> The `StatusBar` object provides some functions to customize the iOS and Android StatusBar, as well as some browser chrome.
```

Comment:
What does "some browser chrome" exactly mean here? 
"browser window on some (mobile OS system) browsers"?

Reply:
It varies by browser support of course, but it typically means the tab color and the navigation bar color.

Reply:
I can extend the explanation of the affected browser chrome, but I think that is better handled by [the caniuse page](https://caniuse.com/#search=theme-color).